dev/Breadcrumb/BreadcrumbItem.cpp

```diff
@@ -299,6 +302,8 @@ void BreadcrumbItem::InstantiateFlyout()
     {
         // Create ItemsRepeater and set the DataTemplate 
         const auto& ellipsisItemsRepeater = winrt::ItemsRepeater();
+        ellipsisItemsRepeater.Name(L"PART_EllipsisItemsRepeater");
+        winrt::AutomationProperties::SetName(ellipsisItemsRepeater, L"EllipsisItemsRepeater");
```

Comment:
This looks very strange, why are we setting an AutomationName from code behind in a keyDownEvent?